stable/cosi/values.yaml

```
@@ -0,0 +1,8 @@
+image:
+  registry: gcr.io
+  repository: k8s-staging-sig-storage/objectstorage-controller
+  tag: v20230915-v0.1.1-33-g9ad2990
```

```suggestion
  tag: v20241219-v0.1.2-alpha1-36-g2c87d5a
```
See https://jira.nutanix.com/browse/NCN-105118

---

Good call, updated to an image from a few weeks ago that has most of these fixed.